server/public/model/group.go

```diff
@@ -52,18 +52,30 @@ func (group *Group) Auditable() map[string]interface{} {
 	return map[string]interface{}{
 		"id":              group.Id,
 		"source":          group.Source,
-		"remote_id":       group.RemoteId,
+		"remote_id":       SafeDereference(group.RemoteId),
 		"create_at":       group.CreateAt,
 		"update_at":       group.UpdateAt,
 		"delete_at":       group.DeleteAt,
 		"has_syncables":   group.HasSyncables,
-		"member_count":    group.MemberCount,
+		"member_count":    SafeDereference(group.MemberCount),
 		"allow_reference": group.AllowReference,
 	}
 }
 
 func (group *Group) LogClone() any {
-	return group.Auditable()
+	return map[string]interface{}{
+		"id":              group.Id,
+		"name":            group.GetName(),
+		"display_name":    group.DisplayName,
+		"source":          group.Source,
+		"remote_id":       SafeDereference(group.RemoteId),
+		"create_at":       MillisToString(group.CreateAt),
```

Comment:
I don't think we should change the format for these standard field names.  Can we instead have different field names for human readable timestamps?

Reply:
Should we discuss the time format changes in https://github.com/mattermost/mattermost/pull/26860#discussion_r1585528053 first?